stdlib/curses/__init__.pyi

```diff
@@ -1,5 +1,5 @@
 from _curses import *
-from _curses import _CursesWindow as _CursesWindow
+from _curses import window as window
```

Comment:
Do we need the `as` import? `window` still appears to be undocumented.

Reply:
Yes, we need this, and this import is kinda the whole point of the PR. Instead of annotating parameters with `foo: curses._CursesWindow`, you can now do a nice and readable `foo: curses.window`.

Reply:
_CursesWindow was re-exported here in https://github.com/python/typeshed/issues/3116 / https://github.com/python/typeshed/pull/3117 for the convenience of `curses._CursesWindow` over `_curses._CursesWindow`.

I don't have an opinion either way, but did the `as` import to keep the same functionality with the new name.